app/views/navigation/dropdown/_profile.html.haml

```diff
@@ -34,6 +34,10 @@
       %i.fa.fa-fw.fa-flask
       = t(".feedback.features")
     .dropdown-divider
+    %a.dropdown-item{ href: "#", data: { bs_target: "#modal-hotkeys", bs_toggle: "modal", hotkey: "Shift+?,?" } }
+      %i.fa.fa-keyboard
+      = t(".hotkeys")
+    .dropdown-divider
```

Comment:
This section of the menu is inside the `if APP_CONFIG["canny"]` condition, please move that outside of that.

Reply:
Oops.